### PR TITLE
enable rust protobuf with bytes feature

### PR DESCRIPTION
**Pull request by overvenus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,12 +19,19 @@ go: link_gopath_src
...
 
 link_gopath_src:
 	rm -f _vendor/src
 	ln -s ./vendor _vendor/src
 
+update_include:
+	mkdir -p proto/include/gogoproto
+	cp _vendor/vendor/github.com/gogo/protobuf/gogoproto/gogo.proto proto/include/gogoproto/
+	cp -r _vendor/vendor/github.com/gogo/protobuf/protobuf/google proto/include
+	curl -# https://raw.githubusercontent.com/stepancheg/rust-protobuf/cee23c63673ad07883c1afd042e8402c171f09a7/proto/rustproto.proto\
```
Comment (siddontang) [inline Makefile]: why use this?

**Comment by siddontang** [comment]
Do we need to update this in grpc-rs project too? 
@BusyJay

**Comment by BusyJay** [comment]
Not necessary as long as the der/ser interfaces are not changed.

File: Makefile (hunk 1 above)
Comment (overvenus) [inline Makefile]: The `#` ? You know, just for fun.

Comment (siddontang) [inline Makefile]: so remove it if we don't need.

**Comment by siddontang** [comment]
LGTM
PTAL @BusyJay

**Pull request closed without merging**